src/pykeen/constants.py

```diff
@@ -70,3 +77,27 @@
 COLUMN_LABELS: Tuple[Target, Target, Target] = (LABEL_HEAD, LABEL_RELATION, LABEL_TAIL)
 TARGET_TO_KEY_LABELS = {target: [c for c in COLUMN_LABELS if c != target] for target in COLUMN_LABELS}
 TARGET_TO_KEYS = {target: [TARGET_TO_INDEX[c] for c in cs] for target, cs in TARGET_TO_KEY_LABELS.items()}
+
+PYKEEN_TEMPORAL_MODULE: pystow.Module = pystow.module("pykeen-temporal")
+#: A path representing the PyKEEN-temporal data folder
+PYKEEN_TEMPORAL_HOME: Path = PYKEEN_TEMPORAL_MODULE.base
+#: A subdirectory of the PyKEEN-temporal data folder for datasets, defaults to ``~/.data/pykeen-temporal/datasets``
+PYKEEN_TEMPORAL_DATASETS: Path = PYKEEN_TEMPORAL_MODULE.join("datasets")
+#: A subdirectory of the PyKEEN-temporal data folder for benchmarks, defaults to ``~/.data/pykeen-temporal/benchmarks``
+PYKEEN_TEMPORAL_BENCHMARKS: Path = PYKEEN_TEMPORAL_MODULE.join("benchmarks")
+#: A subdirectory of the PyKEEN-temporal data folder for experiments,
+# defaults to ``~/.data/pykeen-temporal/experiments``
+PYKEEN_TEMPORAL_EXPERIMENTS: Path = PYKEEN_TEMPORAL_MODULE.join("experiments")
```

Comment:
I would suggest to re-use the existing pykeen directories/`pystow.Module`s directly.